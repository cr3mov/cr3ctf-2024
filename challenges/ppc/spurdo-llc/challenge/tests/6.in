100 193151
5000000 5002899 5001079 5007606 5000000 5000000 5002283 5005614 5000000 5002906 5006464 5000000 5003350 5005511 5003284 5000000 5000000 5005125 5000000 5003912 5002171 3362022 5007431 976278 5002496 5007413 5000000 5006843 5000000 5001191 5006393 5003578 5000000 5000000 5003869 5001180 2552227 5000000 5005435 5001811 5000000 5001199 5001348 5000000 5001074 5002621 5000000 5002784 5000000 5004228 5000028 5000000 5000631 5000000 5000428 5000000 5007373 2774857 5000480 5007616 5006934 5000000 5000000 5000000 5000000 4134110 5000000 1223192 5000000 5000000 969375 5000000 5005789 5000000 5002934 5000000 5005552 5000000 5000000 5004151 5000000 4278321 2432359 5007761 5007542 5000000 5002447 5002089 1758270 5000456 5000000 5006990 2520041 2051525 5004646 5001537 5000000 5004648 5000000 5000000
