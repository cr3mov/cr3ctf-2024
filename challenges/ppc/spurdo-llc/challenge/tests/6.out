4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 4999999 3362022 5000000 976278 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 2552227 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 2774857 5000000 5000000 5000000 5000000 5000000 5000000 5000000 4134110 5000000 1223192 5000000 5000000 969375 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 5000000 4278321 2432359 5000000 5000000 5000000 5000000 5000000 1758270 5000000 5000000 5000000 2520041 2051525 5000000 5000000 5000000 5000000 5000000 5000000 